io/disk/fiotest.py.data/fio_rand_read.yaml

```diff
@@ -0,0 +1,9 @@
+fio_randread.yaml:
+parameters:
+    disk: '/dev/sdd'
```

Comment:
same as above and rest of places, a readme file explaining the values or a comment here should help